python/essex-config/essex_config/configuration_field.py

```diff
@@ -0,0 +1,25 @@
+"""Configuration Field module."""
+
+from dataclasses import dataclass
+from enum import StrEnum, auto
+
+
+class FieldType(StrEnum):
```

Comment:
FieldType may communicate data-type, can we rename this to `classification`?

Reply:
I think that classification suggests the existence a whole taxonomy hehe. Why not FieldVisibility or something along that line?

Reply:
I like FieldVisibility 👍